File: FactoryTs/tipoarticulo.ts

export class TipoArticulo{

    public idTipoArticulo:number;
    public tipoArticulo:string;
    public static TIPO_INFORMATICO:string = "INFORMATICO";
    public static TIPO_OFICINA:string = "OFICINA";

    constructor(idTipoArticulo: number, tipoArticulo:string){
        this.idTipoArticulo = idTipoArticulo;
        this.tipoArticulo = tipoArticulo;
    }

    getIdTipoArticulo():number{
        return this.idTipoArticulo;
    }

    setIdTipoArticulo(idTipoArticulo: number){
        this.idTipoArticulo = idTipoArticulo;
    }

    getTipoArticulo():string{
        return this.tipoArticulo;
    }

    setTipoArticulo(tipoArticulo: string){
        this.tipoArticulo = tipoArticulo;
    }
}

File: FactoryExample/src/factoryexample/TipoArticulo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package factoryexample;

/**
 *
 * @author fabm_
 */
public class TipoArticulo {
    
    private int idTipoArticulo;
    private String tipoArticulo;
    public final static String TIPO_INFORMATICO = "INFORMATICO";
    public final static String TIPO_OFICINA = "OFICINA";
    
    public TipoArticulo(int idTipoArticulo, String tipoArticulo){
        this.idTipoArticulo = idTipoArticulo;
        this.tipoArticulo = tipoArticulo;
    }

    /**
     * @return the idTipoArticulo
     */
    public int getIdTipoArticulo() {
        return idTipoArticulo;
    }

    /**
     * @param idTipoArticulo the idTipoArticulo to set
     */
    public void setIdTipoArticulo(int idTipoArticulo) {
        this.idTipoArticulo = idTipoArticulo;
    }

    /**
     * @return the tipoArticulo
     */
    public String getTipoArticulo() {
        return tipoArticulo;
    }

    /**
     * @param tipoArticulo the tipoArticulo to set
     */
    public void setTipoArticulo(String tipoArticulo) {
        this.tipoArticulo = tipoArticulo;
    }
    
    
    
}
